.azure-pipelines/all.yml

```diff
@@ -1,118 +1,116 @@
 schedules:
```

Comment:
Can you please remove the non relevant changes to this file?

Reply:
Yes, mas eu gerei isso com o `ddev validate ci --fix`.